wbooks/src/App/screens/Home/layout.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Redirect } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+import NavBar from './components/NavBar';
+import Main from './screens/Main';
+
+class Home extends React.Component {
+  componentDidMount() {
+    this.props.onMount();
+  }
+  render() {
+    if (!this.props.isLoggedIn) {
+      return <Redirect to="/login" />;
+    }
+    return (
+      <div>
```

Comment:
If you don't need the div you can return a [fragment](https://reactjs.org/docs/fragments.html).
```js
<>
  <NavBar />
  <Main />
<>
```